### PR TITLE
Add FusionEngine message for turning off or modifying atmospheric delay models

### PR DESCRIPTION
**Pull request by wbrannon**
Modifications
-
- Added `AtmosphericDelayModelConfig`, which contains parameters for the ionospheric model (`iono_delay_model`) and the tropospheric model (`tropo_delay_model`)

### CODE REVIEW

**Review by axlan — APPROVED** [review]
LGTM

File: python/fusion_engine_client/messages/configuration.py [hunk 1]
```diff
@@ -560,6 +568,24 @@ class HardwareTickConfig(NamedTuple):
         "wheel_ticks_to_m" / Float32l,
     )
 
+    class AtmosphericDelayModelConfig(NamedTuple):
+        """!
+        @brief Atmospheric delay model configuration settings.
+
+        See @ref VehicleTickMeasurement.
```
Comment (ryancdougherty) [inline python/fusion_engine_client/messages/configuration.py:575]: Why would we look at VehicleTickMeasurement?

File: python/fusion_engine_client/messages/configuration.py [hunk 2]
```diff
@@ -207,6 +208,13 @@ class NmeaMessageType(IntEnum):
     PQTMVER_SUB = 1204
     PQTMTXT = 1205
 
+class IonoDelayModel(IntEnum):
+    OFF = 0
+    KLOBUCHAR = 1
```
Comment (ryancdougherty) [inline python/fusion_engine_client/messages/configuration.py:213]: Maybe we should add `DEFAULT` to this list. That way we get behaviors like "Off", "On", or "something specific". Maybe I am just saying that to register the thought. In the interest of getting a point release out today, let's not make that change now.

**Review by ryancdougherty — APPROVED** [review]
I had one comment about adding `DEFAULT` states to the enums, which would be useful for users resetting things to "normal" if they ever changed them. But for now, I think this is okay.

File: python/fusion_engine_client/messages/configuration.py (hunk 1 above)
Comment (wbrannon) [reply python/fusion_engine_client/messages/configuration.py:575]: Oops darn copy and paste.

File: python/fusion_engine_client/messages/configuration.py (hunk 2 above)
Comment (axlan) [reply python/fusion_engine_client/messages/configuration.py:213]: We should probably not add default to config options to specific values. We have a planned implementation for "not set" and there's already a way to set a config option to default generally.

Comment (wbrannon) [reply python/fusion_engine_client/messages/configuration.py:213]: So are we planning to _not_ add a default option here then? The way I'll set it to work, if the user doesn't set it at all, then it'll default to using the models we currently use.

Comment (axlan) [reply python/fusion_engine_client/messages/configuration.py:213]: We should not at a default option here. @wbrannon I'm not sure what you mean by "if the user doesn't set it at all". Typically we handle defaults in the default user config json files, so I assume that's what you're talking about?

Comment (wbrannon) [reply python/fusion_engine_client/messages/configuration.py:213]: @axlan yes the way you described it is what I mean. Thanks for clarifying.

**Pull request merged**